_data/site.yaml

```diff
@@ -52,53 +52,142 @@ ga:
 
 # Site Navigation
 primary_navigation:
-  - name: Home
-    url: /
-  - name: Blog
-    url: /blog/
-  - name: Document
-    url: /document/
-  - name: Document with sidenav
-    url: /document-with-sidenav/
-  - name: Document submenu
+  - name: Why cloud.gov?
+    key: overview
     children:
-      - name: Navigation link
-        url: "#main-content"
-      - name: Navigation link
-        url: "#main-content"
-      - name: Navigation link
-        url: "#main-content"
+      - name: Overview
+        url: /docs/overview/what-is-cloudgov/
+      - name: We’re FedRAMP authorized
+        url: /docs/overview/fedramp-tracker/
+      - name: Applications we support
+        url: /docs/overview/portfolio-analysis/
+      - name: Benefits of PaaS
+        url: /docs/technology/responsibilities/
+      - name: Customer stories
+        url: /docs/customer-stories/epa-airnow-gov/
+      - name: Our team
+        url: /docs/overview/cloudgov-team/
+  - name: Get started
+    url: /sign-up/
+  - name: Pricing
+    key: pricing
+    children:
+      - name: Pricing
+        url: /pricing/
+      - name: Try a Free Sandbox
+        url: /docs/pricing/free-limited-sandbox/
+      - name: Quota Costs Billing & Limits
+        url: /docs/pricing/quotas/
+      - name: Support Packages
+        url: /support-packages/
+  - name: Documentation
+    key: docs
+    children:
+      - name: Deploying apps
+        url: /docs/deployment/deployment/
+      - name: Managing apps
+        url: /docs/management/plugins/
+      - name: Managing services
+        url: /docs/services/intro/
+      - name: Managing orgs
+        url: /docs/orgs-spaces/dashboard/
+  - name: News
+    url: /news/
+  - name: Knowledge base
+    url: /knowledge-base/
+  - name: Pages
+    url: /pages/
+  - name: Contact us
+    url: /contact/
 
 secondary_navigation:
-  - name: Secondary link
-    url: "#main-content"
-  - name: Another secondary link
-    url: "#main-content"
+  - name: Platform Status
+    url: https://cloudgov.statuspage.io/
+  - name: Log in
+    url: https://login.fr.cloud.gov/login
+
+documentation_navigation:
+  - name: "Overview"
+    identifier: "overview"
+    icon: fa-home
```

Comment:
if you remap these to the USWDS icon names, you can keep this general structure. Although i'd rather these came from the front matter of the files or data files in the folders, rather than listed separately